Jenkinsfile

```diff
@@ -19,7 +18,8 @@ pipeline {
         }
         stage('Build CHIME kotekan') {
           steps {
-            sh '''cd build
+            sh '''mkdir chime-build
+                  cd chime-build
                   cmake -DRTE_SDK=/opt/dpdk \
```

Comment:
Do we not have to revert back to
```
cmake -DRTE_SDK=/opt/dpdk-stable-16.11.4/ \
```
?

Reply:
No, we symlinked that so it would work on both CI servers.